packages/dx-grid-core/src/plugins/integrated-summary/computeds.ts

```diff
@@ -97,6 +97,17 @@ export const groupSummaryValues: GroupSummaryValuesFn = (
         row,
         rows: [],
       });
+      levelIndex = getLevelIndex(levelKey);
+    }
+    const isCollapsedNestedGroupRow = collapsedRows && levelIndex > 0;
+    if (!levelKey || isCollapsedNestedGroupRow) {
+      levels.forEach((level) => {
+        if (!levelKey) {
+          level.rows.push(row);
```

Comment:
Why is array modified in place here and recreated in the `else` case?

```js
level.rows.push(...collapsedRows);
```

Reply:
Agreed, that would be more consistent